pkg/hwinfo/hwinfo.go

```diff
@@ -0,0 +1,13 @@
+package hwinfo
+
+func Inventory() (map[string]interface{}, error) {
+	hw, err := fetchInventory()
+	if err != nil {
+		return nil, err
+	}
+
+	res := make(map[string]interface{})
+	res[""] = hw
```

Comment:
index `""`?

Reply:
Yah, looks ugly )
Reason because it resulting json output should be in form
```
hw.inventory = {
    cpu.0.Manufacture = ""
}
```

but not
```
hw.inventory.cpu.0.Manufacture = ""
```

Probably `MeasurementsMap` needs some sort of
```
func (mm MeasurementsMap) AddInnerWithPrefix(prefix string, m MeasurementsMap) MeasurementsMap {
        mm[prefix] = m
	return mm
}
```